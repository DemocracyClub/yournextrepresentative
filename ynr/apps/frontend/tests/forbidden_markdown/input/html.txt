<dl>
  <dt>Definition list</dt>
  <dd>Is something people use sometimes.</dd>

  <dt>Markdown in HTML</dt>
  <dd>Does *not* work **very** well. Use HTML <em>tags</em>.</dd>
</dl>
