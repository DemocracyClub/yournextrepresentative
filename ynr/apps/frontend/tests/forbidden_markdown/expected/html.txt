<p>&lt;dl&gt;<br>
&lt;dt&gt;Definition list&lt;/dt&gt;<br>
&lt;dd&gt;Is something people use sometimes.&lt;/dd&gt;</p>
<p>&lt;dt&gt;Markdown in HTML&lt;/dt&gt;<br>
&lt;dd&gt;Does <em>not</em> work <strong>very</strong> well. Use HTML &lt;em&gt;tags&lt;/em&gt;.&lt;/dd&gt;<br>
&lt;/dl&gt;</p>
